### PR TITLE
Add file for GitHub Student

### PR DESCRIPTION
**Pull request by githubstudent**
- How did we notice this problem?
- Discuss our solution

screen shot 2015-10-07 at 11 27 45 am

### CODE REVIEW

**Comment by githubstudent** [comment]
@githubteacher can you please review this pull request?

File: githubstudent.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# GitHub Student
+
+I am GitHub Student.
```
Comment (githubteacher) [inline githubstudent.md]: Maybe you should include something about where you’re from?

**Comment by githubteacher** [comment]
This looks good. :+1:

**Comment by juliuspipo** [comment]
Hi! :+1:

**Comment by Virsalus** [comment]
:+1: (this is pretty cool stuff)

**Pull request merged**